leetcode/MDGenerator/LeetCodeSourceData.java

```diff
@@ -0,0 +1,83 @@
+package com.company;
```

Comment:
Your Pull Request title: Add: MD Generator

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']
* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']